src/EditorFeatures/Core.Wpf/InlineRename/UI/Adornment/RenameFlyout.xaml.cs

```diff
@@ -52,6 +58,17 @@ public RenameFlyout(RenameFlyoutViewModel viewModel, IWpfTextView textView, IWpf
             }
         }
 
+        private async Task DismissToolTipsAsync()
+        {
+            var infoSession = _asyncQuickInfoBroker.GetSession(_textView);
+            if (infoSession is null)
+            {
+                return;
+            }
+
+            await infoSession.DismissAsync().ConfigureAwait(false);
```

Comment:
If I read the code correctly, we are dismissing the QuickInfo when rename is triggered as well as returning no QuickInfo item when there's an active rename session.

Reply:
Correct. That's required because dismissing the active session alone doesn't prevent another one being created. We can't actively turn off QuickInfo altogether, so our workaround is to return no information.

Reply:
might be worth documenting.